src/components/ERC20Transfers/utils.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+export { default } from "./ERC20Transfers";
+
+export const fetchNftData = async () => {
+  try {
+    const response = await axios.get("https://api.polygonscan.com/api", {
+      params: {
+        module: "account",
+        action: "tokentx",
+        address: "0x052D1f06cC98AAe6c4D2e9D06c286143E22dB612",
+        startblock: 0,
+        endblock: 99999999,
```

Comment:
@karangorania, we don't have to define and endblock, do we?

Reply:
no, we don’t 

if we remove this start-block and end-block it will show all the transactions.

Should I removed or ?

Reply:
Yes, let's remove those parameters

Reply:
- Done ✅